### DISCUSSION TITLE
ReactSelect on Mantine 7

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#5496] [Q&A]
i use react-select on mantine 7
but it hidden in Card Component

![Screenshot 2566-12-31 at 17 21 11](https://github.com/mantinedev/mantine/assets/12431734/757639ee-3606-4331-88ba-2fa4e53c9832)

my code

const stylesBase = {
  control: (base: any) => ({
    ...base,
    fontSize: "0.88rem",
    minHeight: "36px",
  }),
  menu: (base: any) => ({
    ...base,
    fontSize: "0.88rem",
    zIndex: 9999
  })
};

<ReactSelect options={options} styles={stylesBase}

but if use on mantine 6 it's not hidden on card when set zIndex: 9999

how to fix on mantine 7

**Comment by rtivital** [comment]
Use Paper instead of Card or set overflow to visible in Card root element.

**Reply by goodlythink** [reply]
it's work thank you very much
